src/main/scala/no/ndla/audioapi/AudioApiProperties.scala

```diff
@@ -40,7 +40,7 @@ object AudioApiProperties extends LazyLogging {
   val SearchServer = propOrElse("SEARCH_SERVER", "http://search-audio-api.ndla-local")
   val SearchRegion = propOrElse("SEARCH_REGION", "eu-central-1")
   val RunWithSignedSearchRequests = propOrElse("RUN_WITH_SIGNED_SEARCH_REQUESTS", "true").toBoolean
-  val SearchIndex = "audios"
+  lazy val SearchIndex = propOrElse("SEARCH_ALIAS", "audios")
```

Comment:
@tmg Joa. Men fungerer det der? Nå vil jo testkoden forholde seg til en index som heter `integration-test-index`, mens koden forholder seg til en index som heter `audios`.
Jeg kjøper forøvrig @kstigen s kommentar om at vi ikke har lyst til å blande integrasjonstestdata med "kjørende" data. Så med mindre jeg tar feil og det du har gjort der faktisk fungerer, ser jeg behovet for å la en property styre navnet.
Men i så fall:
- Må den være `lazy`, eller kan den lastes up-front som alle de andre propertyene?
- Og kan vi kalle den noe litt mer beskrivende enn `SEARCH_ALIAS`? F.eks. `SEARCH_INDEX_NAME` el.n.sånt.